gnomad_qc/v3/resources/release.py

```diff
@@ -1,13 +1,19 @@
-V3_RELEASE_VERSION = 'r3.0'
+from gnomad.resources.resource_utils import TableResource
 
-qual_hists_json_path = 'gs://gnomad/release/3.0/json/gnomad.genomes.r3.json'
+from gnomad_qc.v3.resources.constants import CURRENT_RELEASE
+
+qual_hists_json_path = "gs://gnomad/release/3.0/json/gnomad.genomes.r3.json"
```

Comment:
Does this need to parameterized with a version similar to other paths? (Is this going to get overwritten in a v3.1 pipeline)

Reply:
Good catch, I'll add this!